examples/otlptrace/src/main/java/com/google/cloud/opentelemetry/example/otlptrace/OTLPTraceExample.java

```diff
@@ -63,6 +64,18 @@ private static SpanExporter addAuthorizationHeaders(
       } catch (IOException e) {
         System.out.println("error:" + e.getMessage());
       }
+    } else if (exporter instanceof OtlpGrpcSpanExporter) {
+      try {
+        credentials.refreshIfExpired();
```

Comment:
this will only refresh the first time, right?  After they expire, the exporter will stop working.

Reply:
If that is the case, please add a big warning comment

Reply:
Yes, that is the case, I will add a warning.